core/src/game_state.rs

```diff
@@ -220,6 +220,8 @@ pub struct PropagatedState {
     pub players: Vec<Player>,
     pub observers: Vec<Player>,
     landlord: Option<PlayerID>,
+    gameset_winner_declared: bool,
```

Comment:
Can this be a message rather than a game state?

(if it has to be a game state, we should model it as a more complete enum, rather than as two fields)

Reply:
We can. Logically it makes more sense as well since it should be notification, not state. But that will turn ChatMessage into a notification handler, kind of messy.

Let me use a separate pull request to do it. 

we can decide afterwards.

Reply:
You can add another websocket handler if that's helpful -- this is what we do for game statistics, for example.